### ISSUE TITLE
Folding disabled when screen reader attached

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52673]
Environment Details:
VSCode Version : 1.24.0

Additional Details:
MAS Violated: MAS2.1.1

Repro Steps:
1.Launch VS Code.
2.Open Keyboard Shortcuts.

Actual:
Fold Level 1(Ctrl k+ Ctrl 1), Fold Level 2(Ctrl k+ Ctrl 2), Fold Level 3(Ctrl k+ Ctrl 3), Fold Level 4(Ctrl k+ Ctrl 4), Fold Level 5(Ctrl k+ Ctrl 5), Fold Level 6(Ctrl k+ Ctrl 6), Fold Level 7(Ctrl k+ Ctrl 7) shortcut commands are not working.

Expected:
All the shortcut commands should work.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
User will not be able to use fold, unfold shortcuts.

MAS Reference
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Attachment for Reference: 

![fold level](https://user-images.githubusercontent.com/39204909/41893351-1bdc181c-7939-11e8-9477-7e6021939a2e.jpg)

[A11y VSCode KeyboardShortcuts Fold Level.pptx](https://github.com/Microsoft/vscode/files/2132001/A11y.VSCode.KeyboardShortcuts.Fold.Level.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by AccessibilityTestingTeam-TCS** [comment]
Video is already there in "Attachment for Reference" section, also attached the screenshot. Please refer the video for more clarification.

**Comment by chrmarti** [comment]
There are errors in the JSON that might prevent this from working. JSON is no longer using indentation-based folding, so folding regions are aligned with the actual nesting in the JSON document if I remember correctly.

**Comment by isidorn** [comment]
Folding is disabled when screen reader is detected.
This is a design decision, because none of the users that use screen readers ever requested folding and we do not imagine this scenario in real life.

I would prefer to close this as designed, but leaving up to @alexandrudima and @aeschli to decide
